frontend/src/routes/+page.svelte

```diff
@@ -1,3 +1,33 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+  import { get } from '$utils/fetch';
+  import { _ } from 'svelte-i18n';
+
+  interface Course {
+    name: string;
+    user: {
+      first_name: string;
+    };
+  }
+  let searchQuery = '';
+  let courses: Course[] = [];
+
+  async function getCourses() {
+    const response = await get('/v1/courses');
+    const data = await response.json();
```

Comment:
Instead of converting data into some specific type you could use [generic type](https://www.typescriptlang.org/docs/handbook/2/generics.html):
```ts
const data = await response.json<Course[]>();
```